### PR TITLE
#158423678  Add passenger's profile page

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

Adds passengers' profile page which holds;
 
* All registration details about the passenger. 
* Active/open rides that passenger has requested
* Contact numbers for active drivers
* A past rides log (history)

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-passenger-profile-158423678.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `passenger_profile.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-passenger-profile-158423678`

* Run `cd Ride-My-Way/UI/`

* Open `passenger_profile.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158423678](https://www.pivotaltracker.com/story/show/158423678)

#### Screenshots 

passprof1

### CODE REVIEW

**Comment by ghost** [comment]
code looks good

**Comment by Ogollah** [comment]
I feel like the profile page looks congested.

File: UI/passenger_profile.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      <td>Judas</td>
+      <td>Wendani</td>
+      <td>11-11-2017</td>
+    </tr>
+  </table>
+  <hr>
+  <footer>&copy;RideMyWay 2018.<br><a href="#">Contact Us.</a></footer>
+</div>
+</body>
+</html>
```
Comment (LWanjiru) [inline UI/passenger_profile.html]: add new line here

**Review by LWanjiru — CHANGES_REQUESTED** [review]
@artorious I am curious regarding the `active Rides` section, can a passenger have rides running concurrently? Also, as indicated in the previous PR reviews, please consider refactoring how the page looks and feels. I agree with Ogollah, there seems to be a lot going on on this page. Remember everything doesn't have to fit into one page, it can be done in different pages which would then be linked.

**Comment by artorious** [comment]
@rethes @Ogollah @LWanjiru Your input is much appreciated. I've made changes to improve readability and reduce congestion

passprof

**Pull request merged**